projects/storefrontlib/src/cms-components/user/login/login.component.ts

```diff
@@ -13,9 +13,9 @@ export class LoginComponent implements OnInit {
   constructor(private auth: AuthService, private userService: UserService) {}
 
   ngOnInit(): void {
-    this.user$ = this.auth.getUserToken().pipe(
-      switchMap(token => {
-        if (token && !!token.access_token) {
+    this.user$ = this.auth.isUserLoggedIn().pipe(
```

Comment:
login.component.spec is broken because the isUserLoggedIn is not mocked, therefore saying is not a function